.github/workflows/perf_test.yaml

```
@@ -0,0 +1,31 @@
+name: V2 Performance Test
+
+on:
+  schedule:
+    - cron: "0 18 * * *" # currently run daily on 18:00 UTC; [TODO]: run weekly
+  workflow_dispatch: # run on request
+
+jobs:
+  Performance-Test:
+    runs-on: [self-hosted, linux, x64, dev]
```

To use the CI machine that have CI datasets, use "dmount" label instead of "dev"
```suggestion
    runs-on: [self-hosted, linux, x64, dmount]
```